godotsteam/godotsteam.h

```diff
@@ -2404,10 +645,24 @@ class Steam : public Object,
 	uint32_t writeScreenshot(const PackedByteArray &rgb, int width, int height);
 
 	// Timeline
-	void addTimelineEvent(String icon, String title, String description, int32_t priority, float start_offset, float duration, TimelineEventClipPriority possible_clip);
-	void clearTimelineStateDescription(float time_delta);
+	void setTimelineTooltip(String description, float timeDelta);

```

Comment:
We use snake-case for all the variables.

Reply:
got it! will change the new one according to snake_case. I just copied most of it from original steam interface.